addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.xml

```diff
@@ -23,15 +23,15 @@
                             Discuss
                         </div>
                     </t>
-                    <t t-if="threadViewTopbar.thread and threadViewTopbar.thread.isChannelDescriptionChangeable">
+                    <t t-if="threadViewTopbar.hasDescriptionArea">
                         <div class="o_ThreadViewTopbar_threadDescriptionSeparator flex-shrink-0 mx-2"/>
                         <t t-if="!threadViewTopbar.isEditingThreadDescription">
                             <t t-if="threadViewTopbar.thread.description">
-                                <div class="o_ThreadViewTopbar_threadDescription o_ThreadViewTopbar_editableItem text-truncate px-1" t-att-title="threadViewTopbar.thread.description" t-on-click="threadViewTopbar.onClickTopbarThreadDescription" t-on-mouseenter="threadViewTopbar.onMouseEnterTopbarThreadDescription" t-on-mouseleave="threadViewTopbar.onMouseLeaveTopbarThreadDescription" t-att-class="{ 'o-threadDescriptionEditable': threadViewTopbar.isMouseOverThreadDescription and !messaging.isCurrentUserGuest }">
+                                <div class="o_ThreadViewTopbar_threadDescription o_ThreadViewTopbar_editableItem text-truncate px-1" t-att-title="threadViewTopbar.thread.description" t-on-click="threadViewTopbar.onClickTopbarThreadDescription" t-on-mouseenter="threadViewTopbar.onMouseEnterTopbarThreadDescription" t-on-mouseleave="threadViewTopbar.onMouseLeaveTopbarThreadDescription" t-att-class="{ 'o-threadDescriptionEditable': threadViewTopbar.isMouseOverThreadDescription and threadViewTopbar.thread.isDescriptionEditableByCurrentUser}">
                                     <t t-esc="threadViewTopbar.thread.description"/>
                                 </div>
                             </t>
-                            <t t-if="!messaging.isCurrentUserGuest and !threadViewTopbar.thread.description and threadViewTopbar.thread.isChannelDescriptionChangeable">
+                            <t t-if="!messaging.isCurrentUserGuest and !threadViewTopbar.thread.description and threadViewTopbar.thread.isDescriptionEditableByCurrentUser">
```

Comment:
condition `!messaging.isCurrentUserGuest` could be removed, because already included in the new field `isDescriptionEditableByCurrentUser`